Don't add @princeton.edu to UIDs when they are email addresses already

escowles opened the pull request:
Fixes #1482

tpendragon commented:
Should we have a migration for existing users?

escowles commented:
@tpendragon Yep, just added that as a rake task.

tpendragon commented:
@escowles Can you rebase this? I expect it to fail CI now, since this should be in release notes.

escowles commented:
👍 I rebased and added a release notes entry (also opened https://github.com/pulibrary/figgy/pull/3248 with a straight rebase to make sure it failed, and it does)

tpendragon commented on RELEASE_NOTES.md:
```diff
@@ -8,6 +8,8 @@ Release notes template:
 
 ## Fixed
 
+* Email addresses for external users no longer have "@princeton.edu" added to the end of valid email addresses.
```
I think you added this to the documentation template instead of as a new entry.

escowles replied:
😞 yes, that's exactly what I did.
